src/test/java/dukeapp/parse/ParseTest.java

```diff
@@ -0,0 +1,19 @@
+package dukeapp.parse;
+
+import org.junit.jupiter.api.Test;
+import dukeapp.files.*;
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ParseTest {
+    @Test
+    public void decodeTest_bye() {
```

Comment:
I would suggest having 2 tests to test list and bye separately, and the names to be decodeAction_byeAction_returnTrue and decodeAction_listAction_returnFalse